libs/clients/aircraft-registry/src/lib/apiConfig.ts

```diff
@@ -0,0 +1,33 @@
+import { createEnhancedFetch } from '@island.is/clients/middlewares'
+import {
+  ConfigType,
+  LazyDuringDevScope,
+  XRoadConfig,
+} from '@island.is/nest/config'
+import { Configuration } from '../../gen/fetch'
+import { AircraftRegistryClientConfig } from './aircraftRegistryClient.config'
+
+export const ApiConfig = {
+  provide: 'IcelandicGovernmentInstitutionVacanciesClientConfiguration',
+  scope: LazyDuringDevScope,
+  useFactory: (
+    xroadConfig: ConfigType<typeof XRoadConfig>,
+    config: ConfigType<typeof AircraftRegistryClientConfig>,
+  ) => {
+    return new Configuration({
+      fetchApi: createEnhancedFetch({
+        name: 'clients-aircraft-registry',
+        logErrorResponseBody: true,
+        treat400ResponsesAsErrors: true,
+        timeout: 20000,
```

Comment:
Sure 400 responses be treated as errors? Can skip `logErrorResponseBody` and `timeout` since these are the current default values.